### DISCUSSION TITLE
Inquiry about error occur

### DISCUSSION THREAD

**Discussion by Jimmy-7664** [PaddlePaddle/PaddleSpatial#192] [BaiduKDDCup2022]
Dear officer,
we submitted a trained baseline model GRU. However, I got the following message : [OOMKilled] job failed because of insufficient memory.
Can we get more detail? My team's name is "Kashiwa". The submitted time is 2022-06-19 11:15.
So someone come and help?

Best Regards

**Comment by ramber1836** [comment]
Sorry, we cannot find a team with the name of 'Kashiwa', is 'NOTACTIVE' your team name? 
Anyway, no detailed error message is available if the OOMKilled error is encountered. The max memory of the online test is 16GB, please check the memory cost of your code on your local machine first. 
Thanks.
